src/commands/dev/index.js

```diff
@@ -408,7 +408,7 @@ class DevCommand extends Command {
           functionWatcher.on('change', functionBuilder.build)
           functionWatcher.on('unlink', functionBuilder.build)
         }
-        const functionsPort = await getPort({ port: settings.functionsPort || 34567 })
+        const functionsPort = settings.functionsPort || 34567
```

Comment:
```suggestion
        const functionsPort = settings.functionsPort
```
Probably shouldn't hardcode the default here since it's already in `serve-functions.js`:
https://github.com/netlify/cli/blob/6fe0d2aa49197148804b5292e01fed09621c55f9/src/utils/serve-functions.js#L15
https://github.com/netlify/cli/blob/6fe0d2aa49197148804b5292e01fed09621c55f9/src/utils/serve-functions.js#L194
Otherwise `defaultPort` would never be used.